code/modules/antagonists/vampire/powers/presence/awe.dm

```diff
@@ -0,0 +1,192 @@
+				owner.face_atom(object_of_desire)
+				to_chat(owner, span_awe("Your knees feel wobbly."), type = MESSAGE_TYPE_INFO)
+
+				owner.apply_damage(rand(10,30), STAMINA, owner.get_bodypart(BODY_ZONE_L_LEG), FALSE, TRUE)	// left
+				owner.apply_damage(rand(10,30), STAMINA, owner.get_bodypart(BODY_ZONE_R_LEG), FALSE, TRUE)	// right
+		if(6)
+			if(!owner.incapacitated(IGNORE_RESTRAINTS))
+				owner.face_atom(object_of_desire)
+				to_chat(owner, span_awe("You should move closer."), type = MESSAGE_TYPE_INFO)
+				// Step towards them, but not if that would swap places with them. This feels bad but oh well.
+				if(owner.body_position == STANDING_UP && get_step(owner.loc, get_dir(owner.loc, object_of_desire.loc)) != object_of_desire.loc)
+					owner.visible_message(span_warning("[owner] stumbles dumbly towards [object_of_desire]."), span_awe("You stumble towards [object_of_desire]."))
+					owner.Move(get_step(owner.loc, get_dir(owner.loc, object_of_desire.loc)))
+		if(7)
+			if(!owner.incapacitated(IGNORE_RESTRAINTS))
+				owner.face_atom(object_of_desire)
+				to_chat(owner, span_awe("You should be kind to [object_of_desire]."), type = MESSAGE_TYPE_INFO)
+				owner.emote("smiles")
+		if(8 to 10)
+			return
+
+/datum/status_effect/awed/on_apply()
+	if(!iscarbon(owner))
+		return FALSE
+	if(!owner.has_quirk(/datum/quirk/monochromatic) && strong)
+		owner.add_client_colour(/datum/client_colour/glass_colour/pink)
+
+	owner.add_emitter(/obj/emitter/hearts, "awed")
+	return TRUE
+
+/datum/status_effect/awed/on_creation(mob/living/new_owner, target)
+	. = ..()
+	object_of_desire = target
+	if(strong)
+		linked_alert.desc = "Everything else blurs; <b>[object_of_desire]</b> holds my focus and makes [object_of_desire.p_their()] opinion feel 'right'. <b>Though I could ignore it, should there be danger to myself or others...</b>"
+	else
+		linked_alert.desc = "A warm insistence draws me to <b>[object_of_desire]</b>, making [object_of_desire.p_their()] words and presence feel unusually important. <b>Though I could ignore it, should there be danger to myself or others...</b>"
+	return
+
+/datum/status_effect/awed/on_remove()
+	if(!owner.has_quirk(/datum/quirk/monochromatic))
+		owner.remove_client_colour(/datum/client_colour/glass_colour/pink)
+	owner.remove_emitter("awed")
+
+/datum/status_effect/awed/get_examine_text()
+	return span_warning("[owner.p_They()] look[owner.p_s()] awestruck, staring at [object_of_desire].")
+
+/obj/emitter/hearts
+	alpha = 225
+	particles = new/particles/hearts
+
+/particles/hearts
+	color = generator("color", "#ff2a4e", "#ff93fb", UNIFORM_RAND)
+	spawning = 0.05
+	count = 2
+	lifespan = 30
+	fade = 5
+	position = generator("vector", list(-3,6,0), list(3,6,0), NORMAL_RAND)
+	gravity = list(0, 0.2, 0)
+	color_change = 0
+	friction = 0.2
+	drift = generator("vector", list(0.25,0,0), list(-0.25,0,0), UNIFORM_RAND)
+	icon = 'icons/effects/particles/misc.dmi'
+	icon_state = "heart"
+	#ifndef SPACEMAN_DMM
+	fadein = 10
+	#endif
```

Comment:
This needs to completely change, I don't want an ability that forces you to fall in love with someone. The effect should be like a painful obsession on the level of mind control that you struggle to resist; although this should probably be an ability that just sends you mad.